migrations/V181__System_Intake_Contract_Number_Migration.sql

```diff
@@ -0,0 +1,21 @@
+INSERT INTO system_intake_contract_numbers
+(
+    id,
+    system_intake_id,
+    contract_number,
+    created_by,
+    created_at,
+    modified_by,
+    modified_at
+)
+SELECT
+    gen_random_uuid(),
+    si.id,
+    si.contract_number,
```

Comment:
Would it make sense to do this here as well?
```suggestion
    TRIM(si.contract_number),
```

Reply:
Yeah! Great call! Thanks Clay